actions/project.go

```diff
@@ -49,43 +49,80 @@ func DownloadTemplate(c *cli.Context) {
 		log.Fatal("destination not set")
 	}
 
-	repoURL := c.String("r")
+	url := c.String("u")
 
+	err := DownloadFromURLThenExtract(url, destination)
+	if err != nil {
```

Comment:
Can this use the error checking module? - was created to avoid having the code littered with this `if statement`